conf/igenomes.config

```diff
@@ -428,5 +428,9 @@ params {
             readme      = "${params.igenomes_base}/Sus_scrofa/UCSC/susScr3/Annotation/README.txt"
             mito_name   = "chrM"
         }
+        'HPV16' {
+            fasta       = "https://raw.githubusercontent.com/broadinstitute/CTAT-VirusIntegrationFinder/master/testing/HPV16.fa"
+            gtf         = "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.gtf"
+        }
```

Comment:
This is not in igenomes

Reply:
I can't remember why we added this. Maybe for testing before we understood what it was trying to do. It'll be removed.

Reply:
I honestly have no issue having these files.
But if it's for testing, I think they'd better be in a test profile.
For me the whole point of using this `igenomes.config` file, is that we don't have to re-download the files each time.
This is not working there...